src/app/components/all-products-components/all-products.component.ts

```diff
@@ -1,26 +1,27 @@
-import {Component, Input, OnInit} from '@angular/core';
-import {AllProductsService} from "../../services/all-products/all-products.service";
-import {ProductCard} from "../../interfaces/product-card";
-import {ProductCardComponent} from "./all-products-content/product-card/product-card.component";
+import { Component, OnInit } from '@angular/core';
+import { ProductCard } from '../../interfaces/product-card';
+import { AllProductsService } from '../../services/all-products/all-products.service';
+import { FiltersService } from '../../services/all-products/filters/filters.service';
 
 @Component({
   selector: 'app-all-products',
   templateUrl: './all-products.component.html',
   styleUrls: ['./all-products.component.css']
 })
 export class AllProductsComponent implements OnInit {
+
   public products: ProductCard[] = [];
-  public productsQuantity?:number;
+  public productsQuantity?: number;
 
-  constructor(private allProductService: AllProductsService) {
+  constructor(private allProductService: AllProductsService, private filtersService: FiltersService) {
```

Comment:
please wrote like this => constructor(
   private allProductService: AllProductsService,
   private filtersService: FiltersService
   ){}

Reply:
+